.gitignore

```diff
@@ -3,4 +3,6 @@ data.sqlite
 __pycache__/
 *.py[cod]
 *$py.class
-*.pyc
\ No newline at end of file
+*.pyc
+data/
+.DS_Store
```

Comment:
I'm pretty sure `.DS_Store` will show up in other sub-directories and this only solves for the root directory.